triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -875,6 +979,22 @@ private[lf] class Runner private (
           "state" -> triggerUserState(state, trigger.defn.level),
           "message" -> messageVal.value,
         )
+        if (trigger.defn.level == Trigger.Level.High) {
```

Comment:
As these metrics are derived from high-level trigger data structures, they are not collected for low level triggers